redisinsight/ui/src/packages/clients-list/src/index.html

```diff
@@ -5,8 +5,8 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Client list plugin</title>
-  <link rel="stylesheet" href="../../../../../../vendor/global_styles.css" type="text/css" />
-  <link rel="stylesheet" href="../../../../../../vendor/dark_theme.css" type="text/css" />
+  <!-- <link rel="stylesheet" href="../../../../../../vendor/global_styles.css" type="text/css" /> -->
+  <!-- <link rel="stylesheet" href="../../../../../../vendor/dark_theme.css" type="text/css" /> -->
```

Comment:
sorry, I don't understand what might be affected by this. Was it forgotten? Are you sure?

Reply:
It's one of the reasons why the build plugins sometimes are failed.